spring-cloud-alibaba-nacos-config/src/main/java/org/springframework/cloud/alibaba/nacos/refresh/NacosContextRefresher.java

```diff
@@ -108,11 +119,16 @@ public void receiveConfigInfo(String configInfo) {
 								.toString(16);
 					}
 					catch (NoSuchAlgorithmException | UnsupportedEncodingException e) {
-						logger.warn("[Nacos] unable to get md5 for dataId: " + dataId, e);
+						LOGGER.warn("[Nacos] unable to get md5 for dataId: " + dataId, e);
 					}
 				}
 				refreshHistory.add(dataId, md5);
-				contextRefresher.refresh();
+				applicationContext.publishEvent(
+						new RefreshEvent(this, null, "Refresh  Nacos config"));
```

Comment:
duplicate blank

Reply:
Thank you for your review,I have fix it .